usvm-ts/src/test/kotlin/org/usvm/util/TSMethodTestRunner.kt

```diff
@@ -172,6 +173,8 @@ open class TSMethodTestRunner : TestRunner<TSTest, MethodDescriptor, EtsType?, T
                 TSObject.TSNumber.Double::class -> EtsNumberType
                 TSObject.TSNumber.Integer::class -> EtsNumberType
                 TSObject.UndefinedObject::class -> EtsUndefinedType
+                TSObject.Object::class -> EtsUnknownType
+                TSObject::class -> EtsUnknownType
```

Comment:
Look suspicious

Reply:
It's alright, ```TSObject::Object``` is typescript obj representation. ```TSObject``` is for untyped tests, since we don't want to limit which objects we expect to serialize from models because of type coercion.

Reply:
It is a sealed interface, how we have a situation when we have something different from already been analyzed? For me it looks like an error